Makefile

```diff
@@ -1,21 +1,32 @@
-SKIP_SQUASH?=0
-VERSIONS="2.4.41"
+OPENLDAP_VERSION := $(shell git describe --tags --always --dirty)
+DOCKER_REGISTRY := dockerhub.com
+DOCKER_IMAGE_NAME := openshift/openldap
+DOCKER_IMAGE := $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME)
+TIMESTAMP_RFC3339 := $(shell date +%Y-%m-%dT%T%z)
```

Comment:
seems unused

Reply:
I considered to add a label with a timestamp in it. Do you think that would be useful? Otherwise I throw it out.

Reply:
if you find a fitting label, use it, otherwise throw it away :slightly_smiling_face: